pyveda/training.py

```diff
@@ -324,35 +324,39 @@ class VedaCollection(BaseSet):
           tilesize (tuple): the shape of the imagery stored in the data. Used to enforce consistent shapes in the set.
           partition (str):internally partition the contents into `train,validate,test` groups, in percentages. Default is `[100, 0, 0]`, all datapoints in the training group.
     """
-    def __init__(self, name, mlType="classification", tilesize=[256,256], partition=[100,0,0], **kwargs):
+    def __init__(self, name, mlType="classification", tilesize=[256,256], partition=[100,0,0],
+                imshape = None, dtype = None, percent_cached = 0, sensors = [], _count = 0,
+                _datapoints = None, id = None, image_refs = None,classes = [], bounds = None,
+                user_id = None, public = False, host = HOST, links = None):
+
         assert mlType in valid_mltypes, "mlType {} not supported. Must be one of {}".format(mlType, valid_mltypes)
         super(VedaCollection, self).__init__()
         #default to 0 bands until the first load
-        if 'imshape' in kwargs:
-            self.imshape = tuple(map(int, kwargs['imshape']))
+        if 'imshape':
+            self.imshape = tuple(map(int, imshape))
         else:
             self.imshape = [0] + list(tilesize)
         self.partition = partition
-        self.dtype = kwargs.get('dtype', None)
+        self.dtype = dtype
         if self.dtype is not None:
             self.dtype = np.dtype(self.dtype)
-        self.percent_cached = kwargs.get('percent_cached', 0)
-        self.sensors = kwargs.get('sensors', [])
-        self._count = kwargs.get('count', 0)
-        self._datapoints = None
-        self.id = kwargs.get('id', None)
+        self.percent_cached = percent_cached
+        self.sensors = sensors
+        self._count = _count
+        self._datapoints = _datapoints
```

Comment:
Looks like this is the one thing that wasn't getting set from init args kwargs. There's prob a reason for that, so let's keep it set to the original self._datapoints=None